react-basic-practice/src/pages/dashboard.tsx

```diff
@@ -15,17 +15,17 @@ import { MODAL_TITLES } from "@constants";
 const Dashboard: React.FC = () => {
   const [isOpen, setIsOpen] = useState(false);
   const [modalTitle, setModalTitle] = useState("Add Customer");
-  const [viewData, setViewData] = useState<ICustomer | undefined>(undefined);
+  const [customerData, setCustomerData] = useState<ICustomer | undefined>(undefined);
```

Comment:
remove undefined type. Default it already undefined type

Reply:
i done